### ISSUE TITLE
some issue about this flappy bird

### ISSUE THREAD

**Issue by bupt-steven** [shalabhsingh/A3C_Keras_FlappyBird#2]
Excuse me?
     I have some problem about this project . My OS is the Ubuntu 16.04 ,but I find an error that called 
    
      File "game/flappy_bird_utils.py", line 43, in load
     SOUNDS['die']    = pygame.mixer.Sound('assets/audio/die' + soundExt)
     pygame.error: Unable to open file 'assets/audio/die.ogg'

    I am  sure that is not the path problem( I checked), so Can you tell me your environments about os and something?

**Comment by shalabhsingh** [comment]
This was originally made in windows 10 environment. However, on your request I just checked it in a Ubuntu 16.04 machine and it was working fine over there as well.

Maybe you are running a different python file. Please refer to **How to Run?** section in readme for how to use this repository.

In case you get this error again, can you tell which file you are running and what is the complete stack-trace that you have obtained?

**Comment by bupt-steven** [comment]
firstly, Thanks for your reply.
And I tried it on Windows 10 again, It ran ok.

whatever I ran the test.py or train_network.py on my ubuntu 16.04 , they all told me this error, I guessed it maybe caused by my Graphic Driver .

In actually, I find that there are Actor, Critic, Agent and central Agent , some param is Entropy_rate (maybe is equal your Beta, The RL will let it change from 1 to 0.01) in  other A3C's code, let me  study your code again.
